content/en/tracing/profiler/search_profiles.md

```diff
@@ -140,7 +140,7 @@ Allocations
 : Shows the number of objects allocated by each function in heap memory during the profiling period (default: 60s), including allocations which were subsequently freed. Go calls this `alloc_objects`. Stack allocations are not tracked. This is useful for investigating garbage collection load. See also the note about how this measure changes in version `1.33.0` in [Delta profiles](#delta-profiles).
 
 Allocated Memory
-: Shows the amount of heap memory allocated by each function during the profiling period (default: 60s), including allocations which were subsequently freed. Go calls this `alloc_space`. Stack allocations are not tracked. This is useful for investigating garbage collection load. See also the note about how this measure changes in version `1.33.0` in [Delta profiles](#delta-profiles).
+: Shows the amount of heap memory allocated by each function during the profiling period (default: 60s), including allocations which were subsequently freed. Go calls this `alloc_space`. Stack allocations are not tracked. This is useful for investigating garbage collection load and total memory usage. See also the note about how this measure changes in version `1.33.0` in [Delta profiles](#delta-profiles).
```

Comment:
> and total memory usage

NIT: I think this addition would be a better fit for the `Heap Live Size` profile. Allocation rate and memory usage are not always correlated IMO.

Reply:
Whoops! Bad edit, nice catch :-)